Demo/src/main/java/com/paypal/android/uishared/components/StepHeader.kt

```diff
@@ -0,0 +1,77 @@
+package com.paypal.android.uishared.components
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Surface
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.PlatformTextStyle
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.paypal.android.utils.UIConstants
+
+@Composable
+fun StepHeader(
+    stepNumber: Int,
+    title: String,
+    modifier: Modifier = Modifier
+) {
+    Row(
+        verticalAlignment = Alignment.CenterVertically,
+        modifier = modifier
+    ) {
+        Box(
+            modifier = Modifier
+                .background(
+                    color = MaterialTheme.colorScheme.inverseSurface,
+                    shape = CircleShape
+                )
+                .size(40.dp)
```

Comment:
Would it be worth adding the `40.dp` to the UIConstants?

Reply:
Totally. We can cut down on the magic numbers I agree. Still figuring out the overall semantics of the Compose UI in the Demo app. Updating this will be a good move in the right direction.

Reply:
Cleaned up remaining magic numbers (in [a52845c](https://github.com/paypal/paypal-android/pull/222/commits/a52845c83ae4356df787da376fb11eeb41a28d13)).